packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -604,48 +596,47 @@ const ERRORS = {
         message: 'Mistyped email? %(domain)s does not offer email.'
     },
      */
-    IMAGE_TOO_LARGE: {
-        errno: 1065,
-        message: 'The image file size is too large to be uploaded.'
-    },
-    ECOSYSTEM_ANON_ID_UPDATE_CONFLICT: {
-        errno: 190,
-        message: 'Could not update Ecosystem Anon ID because precondition failed.', // only used for logging, no need for translation
-    },
+  IMAGE_TOO_LARGE: {
+    errno: 1065,
+    message: 'The image file size is too large to be uploaded.',
+  },
+  ECOSYSTEM_ANON_ID_UPDATE_CONFLICT: {
+    errno: 190,
+    message: 'Could not update Ecosystem Anon ID because precondition failed.', // only used for logging, no need for translation
+  },
 };
 
 type ErrorKey = keyof typeof ERRORS;
 type ErrorVal = { errno: number; message: string };
 
-export const AuthUiErrors: { [key in ErrorKey]: AuthUiError } = (Object.entries(
-    ERRORS
-) as [[ErrorKey, ErrorVal]]).reduce(
-    (acc: { [key in ErrorKey]: AuthUiError }, [k, v]) => {
-        const e = new Error(v.message) as AuthUiError;
-        e.errno = v.errno;
-        acc[k] = e;
-        return acc;
-    },
-    {} as Record<ErrorKey, AuthUiError>
-);
+export const AuthUiErrors: { [key in ErrorKey]: AuthUiError } = (
+  Object.entries(ERRORS) as [[ErrorKey, ErrorVal]]
+).reduce((acc: { [key in ErrorKey]: AuthUiError }, [k, v]) => {
+  const e = new Error(v.message) as AuthUiError;
+  e.errno = v.errno;
+  acc[k] = e;
+  return acc;
+}, {} as Record<ErrorKey, AuthUiError>);
 
-export const AuthUiErrorNumbers: {
-    [key in ErrorKey]: number;
+export const AuthUiErrorNos: {
```

Comment:
I must have added this with incorrect formatting :/. `AuthUiErrorNos` is the only changed function here.